backend/app/main.py

```diff
@@ -92,3 +97,96 @@ async def init_redis():
 @app.get("/")
 async def read_root():
     return {"message": "Welcome to KEAPlan's Web API - Go to /docs for an API overview"}
+
+
+users_db = {
+    "henrikpoelse@stud.kea.dk": {
+        "username": "henrikpoelse6666",
+        "full_name": "John Doe",
+        "password": "123456",
+        "user_type": "student",
+        "person_id": 1,
+        "class_id": 1,
+    },
+    "pubae@stud.kea.dk": {
+        "username": "pubae1234",
+        "full_name": "Bølle Bob",
+        "password": "321",
+        "user_type": "lecturer",
+        "person_id": 4,
+        "class_id": 2,
+    },
```

Comment:
Should these live in the DB or is that a new issue?

Reply:
New issue I'm working on. Should probably create the issue, tbh

Reply:
#31 ?

Reply:
Yep